out/blocks/wd_admin_payment_main_form.tpl

```diff
@@ -141,26 +144,19 @@
           </select>
         [{/if}]
 
+        [{if $configField.type === 'button'}]
+          <input type="button" value="[{oxmultilang ident=$configField.multilangid}]" onclick="[{$configField.onclick}]" />
+        [{/if}]
+
         [{if $configField.description}]
           [{$oViewConf->getInputHelpHtml($configField.description)}]
         [{/if}]
       </td>
-      [{if $configKey === 'countryCode'}]
+      [{if $configField.validation}]
         <td>
```

Comment:
I think this shouldn't be an own `<td>` element, otherwise it will break the table layout when the text is displayed. I would just add the `<span>` after the description.

Reply:
👍